web/app/scripts/util/drop-tooltip.js

```diff
@@ -1,10 +1,11 @@
 // A tooltip based on the "Drop" popup library.
-'use strict';
+import Drop from 'tether-drop';
+import '../app';
 
 angular.module('biggraph').service('dropTooltipConfig', function() {
   this.enabled = true;
 });
-angular.module('biggraph').directive('dropTooltip', function(dropTooltipConfig) {
+angular.module('biggraph').directive('dropTooltip', ['dropTooltipConfig', function(dropTooltipConfig) {
```

Comment:
Import `dropTooltipConfig`!

Reply:
> Import `dropTooltipConfig`!

No need. It's defines right above this line. 😅 Reviewing 20,000 lines was maybe too much yesterday.